File: weha/source/weha/wikianalysis/diff/sentence/Sentence.java
package mo.umac.wikianalysis.diff.sentence;

import mo.umac.wikianalysis.lexer.Token;

public class Sentence {

	public Token[] tokens;
	public int startPos;
	public int endPos;
	public int length;
	
	public Sentence()
	{
		tokens = null;
		startPos = -1;
		endPos = -1;
		length = 0;
	}
}

File: google-diff/source/weha/diff/Sentence.php
<?php

class Sentence {

	public $tokens;
	public $startPos;
	public $endPos;
	public $length;
	
	public function __construct()
	{
		$tokens = null;
		$startPos = -1;
		$endPos = -1;
		$length = 0;
	}
}